src/Leaflet.js

```diff
@@ -1,24 +1,23 @@
 
-import {version} from '../package.json';
-export {version};
+export const version = '1.9.2';

```

Comment:
I don't quite want to hard code this just yet. Let's leave it as it was before for now.

Reply:
I don't like hardcoding the version number either... the point is just that this code is non-standard, JSON files cannot be imported by web browsers because they are not ES modules.

Reply:
This would require us to change our release process as well, which makes the change too bit for a single PR IMHO. All in due time.